### PR TITLE
Use kong 1.0-centos instead of the RC

### PR DESCRIPTION
**Pull request by hutchic**
- switched from kong:1.0rc3-centos to kong:1.0-centos
- updated the README. Kong control plane needs to be running before we do anything else

### CODE REVIEW

File: servicea.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ spec:
         command: [ "watch" ]
         args: [ "echo `date` | ncat serviceb 8080"]
       - name: kong
-        image: mashape/kong-enterprise:mesh
+        image: kong:1.0-centos
```
Comment (hutchic) [inline servicea.yaml]: I guess we could maybe just swap this for `latest`? 🤔

Comment (james-callahan) [reply servicea.yaml]: Yeah. Could just have this as image `kong`.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 setup_minikube:
```
Comment (james-callahan) [inline Makefile]: This whole rule seems dangerous/odd to me.

Comment (hutchic) [reply Makefile]: It absolutely is. I'll call that out in the README and link to relevant minikube manual setup instructions but the convenience method for those that like to live dangerously is p.nice

**Pull request merged**